gradle.properties

```diff
@@ -1,3 +1,3 @@
-release_version=0.1.5
+release_version=0.1.6
```

Comment:
We should prepare the first release which we will support 
```suggestion
release_version=1.0.0
```

Reply:
Preparation:
* Please use custom ...Request of ...Response classes instead of the google.protobuf.Empty class to be able to extend it in future

Reply:
* the `rpc getMessageStreams (google.protobuf.Empty) returns (StringList)` method should returns a Messages method (similar to the Events message) and contains th2 Messages instead of strings

Reply:
* it will change API, but maybe use typed structure of values in filters instead of using FilterParameterType and string valuse